lib/cocoapods/installer/xcode/pods_project_generator/pod_target_installer.rb

```diff
@@ -414,13 +416,13 @@ def create_test_xcconfig_files(test_native_targets, test_resource_bundle_targets
             target.test_specs.each do |test_spec|
               spec_consumer = test_spec.consumer(target.platform)
               test_type = spec_consumer.test_type
-              path = target.xcconfig_path(test_type.to_s)
+              path = target.xcconfig_path("#{test_type.capitalize}-#{test_spec.base_name}")
```

Comment:
there could in theory be multiple test specs with the same base name?

Reply:
true we should figure out something uniq